src/fuzz_test.cpp

```diff
@@ -0,0 +1,30 @@
+#include <unistd.h>
+
+#define main zig_main
+#include "main.cpp"
+#undef main
+
+extern "C" int LLVMFuzzerTestOneInput(const uint8_t* data, size_t size)
+{
+    char tmp_file_name[] = "/tmp/fuzzXXXXXX";
+    int fd = mkstemp(tmp_file_name);
+    if (fd < 0) {
+        perror("Cannot create temporary file");
+        return 0;
+    }
+    const int num_written = write(fd, data, size);
```

Comment:
I'd use `fdopen()` and `fwrite()` since the latter takes care of retrying, but if you want to use `write()`, handle EINTR:
```c
ssize_t n;
do {
    n = write(fd, data, size);
} while (n == -1 && errno == EINTR);
```

Reply:
Good to know.